### PR TITLE
Update logging in web.js, added rollbar logging for sendQueued.js and sendReminders.js, updated readme

### PR DESCRIPTION
**Pull request by robbiepc30**
Updated all unhandled errors in web.js (which is all of them) to be sent to next so that the error handling middleware can take care of all logging.

Added rollbar logging for sendQueued.js and sendReminders.js.

Also updated README.md with correct environment variable name for the twilio account sid.

### CODE REVIEW

File: web.js [hunk 1]
```diff
@@ -40,10 +40,13 @@ app.get('/', function (req, res) {
 
 // Fuzzy search that returns cases with a partial name match or
 // an exact citation match
-app.get('/cases', function (req, res) {
+app.get('/cases', function (req, res, next) {
   if (!req.query || !req.query.q) return res.send(400);
 
   db.fuzzySearch(req.query.q, function (err, data) {
+    if (err) {
+      next(err);
```
Comment (NigelKibodeaux) [inline web.js]: you should do `return next(err)` so it doesn't do all the stuff below

File: web.js [hunk 2]
```diff
@@ -89,13 +92,12 @@ app.post('/sms', askedReminderMiddleware, function (req, res, next) {
         originalCase: JSON.stringify(req.match)
       }, function (err, data) {
         if (err) {
-          rollbar.handleError(err, req);
+          next(err);
```
Comment (NigelKibodeaux) [inline web.js]: same here

File: web.js [hunk 3]
```diff
@@ -129,6 +131,9 @@ app.post('/sms', askedReminderMiddleware, function (req, res, next) {
   db.findCitation(text, function (err, results) {
     // If we can't find the case, or find more than one case with the citation
     // number, give an error and recommend they call in.
+    if (err) {
+      next(err);
```
Comment (NigelKibodeaux) [inline web.js]: here too

**Comment by robbiepc30** [comment]
Ok, thanks for the feedback. I have updated the code to reflect those changes

**Pull request merged**